lib/ddtrace/contrib/rails/framework.rb

```diff
@@ -41,6 +42,7 @@ def self.config_with_defaults
             config[:database_service] ||= "#{config[:service_name]}-#{Contrib::ActiveRecord::Utils.adapter_name}"
             config[:controller_service] ||= config[:service_name]
             config[:cache_service] ||= "#{config[:service_name]}-cache"
+            config[:web_sockets_service] ||= "#{config[:service_name]}-#{Contrib::ActionCable::Ext::SERVICE_NAME}"
```

Comment:
Given the suggested removal of the option, this would become obsolete.

Reply:
Given I need the Rails service name to create the ActionCable service name, how can I get rid of this line?

Reply:
Removed.